docs/cloud/serverless.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Serverless Preview 
```

Comment:
Serverless Database preview

Capital S and D because these describe the functionality. Whereas "preview" is lower-case "P" because it's what's going on.